chia/plotting/manager.py

```diff
@@ -244,6 +244,10 @@ def _refresh_task(self):
                 def plot_removed(test_path: Path):
                     return not test_path.exists() or test_path.parent not in plot_directories
 
+                for path in list(self.no_key_filenames):
```

Comment:
is the `list()` here just to make a copy, to allow mutating the container inside the loop?
I would have expected something like `.clone()` or `.copy()`

Reply:
> is the `list()` here just to make a copy, to allow mutating the container inside the loop?
> I would have expected something like `.clone()` or `.copy()`

Right, the point is making a copy to allow mutating but i agree, `copy()` is more obvious :) See 8a21546.